src/components/views/rooms/EventTile.js

```diff
@@ -50,17 +50,6 @@ var eventTileTypes = {
 
 var MAX_READ_AVATARS = 5;
 
-// Our component structure for EventTiles on the timeline is:
```

Comment:
Why has this been killed?  It's probably the comment I refer to more than any other in the whole project, in terms of trying to remember what the nesting structure is for the react components in the timeline, and where to introduce new ones.  If it has changed, please can we update the comment rather than delete it?